eng/pipelines/templates/steps/npm-release-task.yml

```diff
@@ -36,11 +39,11 @@ steps:
         condition: succeeded()
 
   - ${{ else }}:
```

Comment:
So the intent is to do both now, with the removal of the conditional?

Reply:
I didn't remove a conditional, I simply updated the whitespace so that the stuff is one level below the if. Then I made the `add additional tag` only call if AdditionalTag is actually specified.